elements/bulbs-video/plugins/google-analytics.test.js

```diff
@@ -333,23 +373,57 @@ describe('Google Analytics', () => {
         on: sinon.spy(),
       };
 
-      let googleAnalytics = GoogleAnalytics.init(player, 'videoplayer0');
+      googleAnalytics = GoogleAnalytics.init(player, 'videoplayer0');
+    });
+
+    it('sends an "aderror" event without making any change if no rnd value', () => {
+      eventStub = {
+        tag:
+          'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0&tt=p&t=1125,the-onion,today-now,main,html5&s=main/the-onion/today-now', // eslint-disable-line max-len
+        message: 'Ad Tag Empty',
+      };
+
+      googleAnalytics.onAdError(eventStub);
+      expect(global.ga).to.have.been.calledWith(
+        'videoplayer0.send',
+        'event',
+        'Video:The Onion',
+        'aderror: Ad Tag Empty',
+        'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0&tt=p&t=1125,the-onion,today-now,main,html5&s=main/the-onion/today-now' // eslint-disable-line max-len
+      );
+    });
+
+    it('sends an "aderror" event filtering out rnd value when at the end', () => {
       eventStub = {
         tag:
-          'http://localhost:8080/fixtures/bulbs-video/html5-vast.xml',
+          'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0&tt=p&t=1125,the-onion,today-now,main,html5&s=main/the-onion/today-now&rnd=2768879373', // eslint-disable-line max-len
         message: 'Ad Tag Empty',
       };
 
       googleAnalytics.onAdError(eventStub);
+      expect(global.ga).to.have.been.calledWith(
+        'videoplayer0.send',
+        'event',
+        'Video:The Onion',
+        'aderror: Ad Tag Empty',
+        'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0&tt=p&t=1125,the-onion,today-now,main,html5&s=main/the-onion/today-now' // eslint-disable-line max-len
+      );
     });
 
-    it('sends an "aderror" event', () => {
+    it('sends an "aderror" event filtering out rnd value when sandwiched', () => {
+      eventStub = {
+        tag:
+          'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0&tt=p&t=1125,the-onion,today-now,main,html5&s=main/the-onion/today-now&rnd=2768879373&foo=bar', // eslint-disable-line max-len
```

Comment:
For these long lines, I think it would help to break them up so it's more immediately obvious how this is different than/same as the expected output.